contrib/externs/chrome_extensions.js

```diff
@@ -11214,3 +11214,96 @@ chrome.downloads.DeterminingFilenameEvent.prototype.hasListeners =
  * https://developer.chrome.com/extensions/downloads#event-onDeterminingFilename
  */
 chrome.downloads.onDeterminingFilename;
+
+
+/**
+ * @const
+ * @see https://developer.chrome.com/extensions/declarativeContent
+ */
+chrome.declarativeContent = {};
+
+
+/** @type {string} */
+chrome.declarativeContent.PageStateMatcherInstanceType;
```

Comment:
AFAICT the docs don't actually show any use of any of these `*InstanceType` values.
How would you feel about dropping them?

Reply:
I don't really know if they are used. They are listed in the docs, just like types for [chrome.downloads](https://developer.chrome.com/extensions/downloads#type-FilenameConflictAction).

Reply:
@brad4d I just found this code: https://github.com/dart-gde/chrome.dart/blob/master/lib/gen/declarative_content.dart
Is that the official repository? It looks like.
If so, following the `chrome.downloads` example, these `*InstanceType` should be `@enum {string}` using the `const` names from there, like `DECLARATIVE_CONTENT_PAGE_STATE_MATCHER`.

Reply:
The difference is that in the chrome.downloads docs `FilenameConflictAction` actually appears as the type for an object property. I don't see anything similar for the declarativeContent instance type definitions.

Also, if we're going to define them, they need to be defined like [ViewType](https://github.com/google/closure-compiler/blob/master/contrib/externs/chrome_extensions.js#L2322).
That's especially weird since these have only a single allowed value and they're not used anywhere.

Reply:
I must add that when I defined all the types I used this file as reference:
https://chromium.googlesource.com/chromium/src/+/lkgr/chrome/common/extensions/api/declarative_content.json

Reply:
@brad4d I updated the `*InstanceType` to respect the code.